cli/src/commands/deploy.ts

```diff
@@ -191,6 +207,11 @@ export default class Deploy extends Command {
       const { applicationId, defaultEnv } = project;
       application = await this.lycanClient.deploy(applicationId, defaultEnv, digest, envVars);
     }
-    this.log(`Project successfully deployed! Your project is now available at: ${makeAppLink(application)}`);
+    const jsonApp = { ...application, link: makeAppLink(application) };
```

Comment:
Looking at `makeAppLink`, it uses `terminalLink` whicch on iterm2 behaves in a funny way - it produces escape codes so the link is clickable.

It does not appear when piping into `jq`, but it's still less useful - I think for the json output it would make sense to just call to `getPrimaryURL`, also maybe rename `link` to `primaryURL` (since there is sometimes more than one).

Reply:
Done